contrib/gcppubsub/pkg/controller/channel/reconcile_test.go

```diff
@@ -782,11 +782,10 @@ func makeChannel() *eventingv1alpha1.Channel {
 
 func makeChannelWithFinalizerAndPCSAndAddress() *eventingv1alpha1.Channel {
 	c := makeChannelWithFinalizerAndPCS()
-	// serviceAddress is the address of the K8s Service. It uses a GeneratedName and the fake client
-	// does not fill in Name, so the name is the empty string.
+	// serviceAddress is the address of the K8s Service.
 	c.Status.SetAddress(&apis.URL{
 		Scheme: "http",
-		Host:   fmt.Sprintf(".%s.svc.%s", c.Namespace, utils.GetClusterDomainName()),
+		Host:   fmt.Sprintf("%s-channel-%s.%s.svc.%s", c.Name, c.UID, c.Namespace, utils.GetClusterDomainName()),
```

Comment:
are there any concerns with the length of the name here that might propagate? Put another way, do we then create a dns entry based off of these somewhere along the way?

Reply:
The real code uses a length check, see GenerateFixedName() in utils.go.

Reply:
that is correct, this is just UTs